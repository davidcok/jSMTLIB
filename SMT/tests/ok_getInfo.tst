(get-info :authors)
(get-info :name)
(get-info :version)
(get-info :error-behavior)
