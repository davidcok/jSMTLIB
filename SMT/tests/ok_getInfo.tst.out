(:authors "David R. Cok" )
(:name "test" )
(:version "0.0" )
(:error-behavior continued-execution )
